std_msgs/Header header
float64         distance
float64         ROLL
float64         PITCH
float64         YAW
float64         P
float64         Q
float64         R
float64         GMX
float64         GMY
float64         GMZ
float64         GMT
float64         TMX
float64         TMY
float64         TMZ
float64         TMT
float64         ACS1
float64         ACS2_1
float64         ACS2_2
float64         ACS2_3
float64         ACS2_4